src/utils.jl

```diff
@@ -415,3 +415,63 @@ macro hijack_observable(name)
     end)
 
 end
+
+############################################################
+#                                                          #
+#     Splitting Of LineString / coast lines                #
+#                                                          #
+############################################################
+
+#`LineSplitting` module defines a `split` method for vectors of `LineString` objects.
+#
+#This is needed to fix e.g. coast line displays when lon_0 is not 0 but cutting polygons at lon_0+-180.
+
+module LineSplitting
```

Comment:
> Why is this in a Module?

Seemed cleaner, no downside, easy to revisit later

> does this only work on LineStrings for now, or is there a more generic way?

I don't recall trying it beyond LineStrings but my assumption would be that it will likely get generalized at some point